pkg/sync/kubernetes/ffclientset.go

```diff
@@ -26,6 +25,14 @@ type FeatureFlagClient struct {
 	ns         string
 }
 
+type FFCInterface interface {
+	FeatureFlagConfigurations(namespace string) Interface
```

Comment:
`Interface` could use a more informative name. Hasn't been introduced here so happy to leave it but something to consider

Reply:
Done